### DISCUSSION TITLE
Allow Scopes to have (count) of objects

### DISCUSSION THREAD

**Discussion by djalmaaraujo** [avo-hq/avo#2274] [Feature requests]
This is very important and common: Showing the count of objects in the Scope label. Something like `Projects (234)`. How can we implement this? You could allow the `label` to be customized, dynamic, receiving the `record`, or something else.

<img width="395" alt="image" src="https://github.com/avo-hq/avo/assets/3402/c1489cf8-26bc-4829-a067-e13155ec0283">

Let me know how I can help.

**Comment by adrianthedev** [comment]
Great idea.
I wonder if it's an easy fix or more difficult to achieve.

**Reply by djalmaaraujo** [reply]
I am trying to find out, by looking at the advanced code. I know that the information is there somewhere because we have the "Displaying X items" on the bottom of the page.

I wonder if I can access this by looking at the scope.resource somehow.

But, it looks like since there are different scope to show, a count query will be needed anyway for each scope.

If you have any clue where to start let me know, I will put some time on this in the following weeks.

Looking at the performance level, I think it should be not the default configuration, but something you can customize on the scope class, do you agree?

We could allow the customization of the name passing a lambda, or using self.something.

**Comment by adrianthedev** [comment]
It will be enabled in the next release by https://github.com/avo-hq/avo/pull/2288 and https://github.com/avo-hq/avo-pro/pull/56

[Docs](https://docs.avohq.io/3.0/recipes/display-scope-record-count.html)
